docs/4.4/preview/teleport-application-access.md

```diff
@@ -0,0 +1,207 @@
+    | Docker Image | [SHA256](https://quay.io/repository/gravitational/teleport?tab=tags) | `docker pull quay.io/gravitational/teleport:5.0.0-beta.10` |
+
+=== "Teleport Enterprise Edition"
+
+    | OPERATING SYSTEM | CHECKSUM | DOWNLOAD LINK |
+    |-|-|-|
+    | Linux 32-bit | [SHA256](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-386-bin.tar.gz.sha265) | [teleport-ent-v5.0.0-beta.10-linux-386-bin.tar.gz](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-386-bin.tar.gz) |
+    | Linux 64-bit | [SHA256](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-amd64-bin.tar.gz.sha256) | [teleport-ent-v5.0.0-beta.10-linux-amd64-bin.tar.gz](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-amd64-bin.tar.gz) |
+    | Linux 64-bit (RHEL/CentOS 6.x compatible) | [SHA256](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-amd64-bin.tar.gz.sha256) | [teleport-ent-v5.0.0-beta.10-linux-amd64-bin.tar.gz](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-amd64-bin.tar.gz) |
+    | Linux ARMv7 | [SHA256](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-arm-bin.tar.gz.sha256) | [teleport-ent-v5.0.0-beta.10-linux-arm-bin.tar.gz](https://get.gravitational.com/teleport-ent-v5.0.0-beta.10-linux-arm-bin.tar.gz) |
+    | Linux 64-bit DEB | [SHA256](https://get.gravitational.com/teleport-ent_5.0.0-beta.10_amd64.deb.sha256) | [teleport-ent_v5.0.0-beta.10_amd64.deb](https://get.gravitational.com/teleport-ent_5.0.0-beta.10_amd64.deb) |
+    | Linux 32-bit DEB | [SHA256](https://get.gravitational.com/teleport-ent_5.0.0-beta.10_i386.deb.sha256) | [teleport-ent_v5.0.0-beta.10_i386.deb](https://get.gravitational.com/teleport-ent_5.0.0-beta.10_i386.deb) |
+    | Linux 64-bit RPM | | [teleport-ent-5.0.0-beta.10-1.x86_64.rpm](https://get.gravitational.com/teleport-ent-5.0.0-beta.10-1.x86_64.rpm)
+    | Docker Image | [SHA256](https://quay.io/repository/gravitational/teleport-ent?tab=tags) | `docker pull quay.io/gravitational/teleport-ent:5.0.0-beta.10` |
+
+
+Follow our standard [installation procedure](https://gravitational.com/teleport/docs/installation/).
+
+
+**Define `/etc/teleport.yaml`**
+
+| Variable to replace | Description  |
+|-|-|
+| `nodename` | Name of node Teleport is running on |
+| `auth_token` | Static Join Token |
+| `public_addr` | Public URL and Port for Teleport |
+| `https_key_file` | LetsEncrypt Key File ( Wildcard Cert )  |
+| `https_cert_file` | LetsEncrypt Cert File ( Wildcard Cert ) |
+
+`teleport.yaml` is a configuration file used by Teleport. For this first example, Teleport has been set up using local storage. After this service is running, we'll connect back the Teleport Application Access service.
+
+```yaml
+teleport:
+  nodename: i-083e63d0daecd1315
+  data_dir: /var/lib/teleport
+  auth_token: 4c7e15
```

Comment:
This auth token is way too short to be secure; I replaced all the instances with a longer randomly generated string (just for best practices)

```suggestion
  auth_token: Jl1u0rwar6bqzlg79ou0kYaWHErUqPsr
```